src/cfn-registry.ts

```diff
@@ -24,10 +35,9 @@ export async function describeResourceType(name: string, _version?: string): Pro
         NextToken: token,
         Type: name.endsWith('MODULE') ? 'MODULE' : 'RESOURCE',
         Filters: {
-          Category: 'THIRD_PARTY',
```

Comment:
Does this still work in the public case? I got errors when trying this without 'THIRD_PARTY'

Reply:
Good question @hoegertn ... We need to check. Maybe we have a regression.

If you have some time to take a look that will be great. Otherwise I'll take a look tomorrow.